configs/0xA9DB20F77D646071eDa1c4AC9Dbfe5bD2fcfB5AB.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please update the wallet address to `0xB96b74874126A787720a464EAb3FBD2F35a5D14e`